# Ideally this input would be the depth and color image.
# Untill python3 support the serve input is a string and the
# output it the grasp selected from the GQCNN and the probability of success.
string name
---
geometry_msgs/PoseStamped grasp
float64 q_val
